spec.md

```diff
@@ -245,6 +243,32 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD chose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are valid:
```

Comment:
s/valid/defined/.

Then afterwards:
Service Brokers MAY include additional fields within the response. When adding new fields, it is SHOULD use a unique prefix on the field names to reduce the chances of conflict with future specification defined fields.

I don't think we can, or should , say what a Platform can do with this data since they can ignore all of it if they want.  Also, this "add a prefix" thing might be good for any other spot in the spec where we allow extension json fields.

Reply:
Done!